ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestConfig.java

```diff
@@ -1065,6 +1078,13 @@ public ClientAuth getClientAuthInternal() {
     return getClientAuth(getString(KSQL_INTERNAL_SSL_CLIENT_AUTHENTICATION_CONFIG));
   }
 
+  public List<KsqlResourceExtension> getKsqlResourceExtensions() {
+    if (getString(KSQL_RESOURCE_EXTENSION).isEmpty()) {
+      return Collections.emptyList();
+    }
+    return getConfiguredInstances(KSQL_RESOURCE_EXTENSION, KsqlResourceExtension.class);
```

Comment:
```suggestion
    return getConfiguredInstances(KSQL_RESOURCE_EXTENSIONS, KsqlResourceExtension.class);
```
we can take `KSQL_RESOURCE_EXTENSIONS` as comma separated or we can allow prefix based `ksql.resource.extension.fipsValidator.class=<className>` with `ksql.resource.extension.fipsValidator.enable` etc.